src/components/Banner/components/BannerIcon/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import Icon from '../../../Icon';
+import Progress from '../../../Progress';
+import { Props } from './types';
+
+
+function BannerIcon({
+    className,
+    iconName,
+    isLoading,
+    ...otherProps
+}: Props): ReactElement {
+
+
+    const iconClasses = clsx(
+        'brew-Banner__icon',
+        className,
+    );
+
+    const loading = (<Progress className="brew-Banner__iconLoading" size="medium" variant="circular" />);
+    const icon = (<Icon className="brew-Banner__iconImage" size="large">{iconName}</Icon>);
```

Comment:
I would add two line breaks before this since it's a separate JSX component and break this component into a multiline for readability. Something like...
```
const icon = (
    <Icon className="brew-Banner__iconImage" size="large">
        {iconName}
    </Icon>
);
```